src/components/nodes/sequenceFlow/sequenceFlowConfig.js

```diff
@@ -0,0 +1 @@
+export const expressionPosition = { distance: 0.25, offset: { x: -40, y: -20 } };
```

Comment:
I think this should be commented. The "Label Position" section of https://resources.jointjs.com/tutorial/link-labels describes how these numbers work. Or you can say something like "position label above and near the source of sequence flow".